.travis.yml

```diff
@@ -12,6 +14,17 @@ install:
   - "pip install git+https://github.com/landscapeio/requirements-detector.git@develop"
   - "pip install --use-mirrors --editable ."
 script:
-  nosetests -s --with-coverage --cover-inclusive --cover-package=prospector tests/
+  - "nosetests -s --with-coverage --cover-inclusive --cover-package=prospector tests/"
+  - "mv .coverage .coverage.x"
+  #  install prospector in develop mode, so that we run coverage against the same
+  #  modules that nose has already previously covered.
+  - "python setup.py develop"
```

Comment:
I replaced this with `pip install -e .[with_everything]` which does `python setup.py develop` under the hood. It's not perfectly equivalent (I've seen some odd differences before) but it seems to be working and doing the same as the intended behaviour. Have a look at https://github.com/landscapeio/prospector/commit/6a00d492bbc867a6fdad4100635a637eee04bf99 and https://travis-ci.org/landscapeio/prospector/builds/45835029 to make sure!